packages/backend/src/models/Membership.ts

```diff
@@ -99,19 +101,19 @@ export class Membership {
           (memberStart.getTime() >= startDate.getTime() &&
             memberStart.getTime() <= endDate.getTime())
         ) {
-          return false;
+          return true;
         }
       });
     }
 
-    return true;
+    return false;
   }
```

Comment:
Even if here you have more comparison, you can also simplify it with a single return.